src/identity/manager.rs

```diff
@@ -318,12 +315,16 @@ impl Worker {
                             (CertState::Available(certs), refresh_at)
                         },
                     };
+
+                    // TODO log. also don't block on this before updating cert for user traffic
+                    let _ = self.output_cert(&state).await;
```

Comment:
why shouldn't we block? shouldn't the cert update before we are able to handle more traffic?

Reply:
This code path is used both to fetch certs for user traffic, and the new code writes the certs to the filesystem to use for xDS/CA authentication.